packages/ui5-middleware-cfdestination/lib/cfdestination.js

```diff
@@ -99,6 +101,19 @@ module.exports = async ({ resources, options, middlewareUtil }) => {
 	// the default auth mechanism is set to none but the user can pass an auth method using the options
 	xsappConfig.authenticationMethod = effectiveOptions.authenticationMethod
 
+	// check if destinations exist in .env file as JSON string
+	if (typeof effectiveOptions.destinations === "string" && effectiveOptions.destinations.startsWith("$env:")) {
```

Comment:
I personally would prefer a regex for those kind of checks but this is also good to go!